sessionize/lib/src/iosMain/kotlin/co/touchlab/sessionize/platform/Functions.kt

```diff
@@ -84,6 +105,59 @@ actual fun settingsFactory(): Settings.Factory = PlatformSettings.Factory()
 
 actual fun createUuid(): String = NSUUID.UUID().UUIDString
 
+
+
+@ExperimentalUnsignedTypes
+actual fun createLocalNotification(title:String, message:String, timeInMS:Long, notificationId: Int) {
```

Comment:
I'm not sure if it will be cleaner, but consider creating an expected class for the notification stuff. Or at least a separate file if you end up keeping them as global functions

Reply:
Good ideas. I didn't want to break the existing format but maybe a separate file would be good. I don't know if the expected class would help since there aren't really any held variables besides the delegate or publisher, which are platform specific. Does anyone else have an opinion on this?

Reply:
Yeah, at least a separate file would be nice for readability. Idk if a class is needed/helpful. I've got a nagging concern about how much goes in the global scope and how it impacts testability, but not sure.

Reply:
Moved to their own files